piet-wgsl/shader/pathseg.wgsl

```diff
@@ -182,14 +181,16 @@ fn main(
                 p1 = mix(p1, p0, 1.0 / 3.0);
             }
         }
+        var stroke = vec2<f32>(0.0, 0.0);
```

Comment:
Probably don’t want to backtrack on these :)

Reply:
Heh, I originally had `vec2<f32>()` but decided to put in the actual numbers.